openmrs-client/src/main/java/org/openmrs/mobile/databases/entities/EncounterTypeEntity.java

```diff
@@ -0,0 +1,29 @@
+package org.openmrs.mobile.databases.entities;
+
+import androidx.room.ColumnInfo;
+import androidx.room.Entity;
+
+import com.google.gson.annotations.Expose;
+import com.google.gson.annotations.SerializedName;
+
+import org.openmrs.mobile.models.Link;
+import org.openmrs.mobile.models.Resource;
+
+import java.util.ArrayList;
+import java.util.List;
+
+@Entity(tableName = "encounterTypes")
```

Comment:
```suggestion
@Entity(tableName = "encounter_types")
```
for naming consistency.